### PR TITLE
Update Status View To Use FlexBox

### PR DESCRIPTION
**Pull request by jv075002**
Flex was added and grid was removed

Addresses https://github.com/cerner/terra-core/issues/2618

### CODE REVIEW

File: packages/terra-status-view/src/StatusView.module.scss [hunk 1]
```diff
@@ -20,25 +15,27 @@
...
 
   .inner-view {
     align-items: center;
     display: flex;
     flex-direction: column;
-    grid-area: content;
     max-height: 100%;
     max-width: 100%;
-    overflow: auto;
+    overflow: unset;
```
Comment (yuderekyu) [inline packages/terra-status-view/src/StatusView.module.scss]: Oops, we actually want to keep overflow within the inner-view. Let's revert this change. Here's some discussion on this https://github.com/cerner/terra-core/issues/2737

File: packages/terra-status-view/src/terra-dev-site/test/status-view/StatusViewTestCommon.module.scss [hunk 2]
```diff
@@ -7,5 +7,6 @@
   .status-view-wrapper-2 {
     border: 1px solid #000;
     width: 75px;
+    height: 500px;
```
Comment (yuderekyu) [inline packages/terra-status-view/src/terra-dev-site/test/status-view/StatusViewTestCommon.module.scss]: we have a lint error here that needs to be fixed https://travis-ci.com/cerner/terra-core/jobs/259205761#L314

File: packages/terra-status-view/CHANGELOG.md [hunk 3]
```diff
@@ -3,8 +3,13 @@ ChangeLog
 
 Unreleased
 ----------
+<<<<<<< HEAD
```
Comment (emilyrohrbough) [inline packages/terra-status-view/CHANGELOG.md]: Oops. Looks like there are some missed merge conflicts

Comment (jv075002) [reply packages/terra-status-view/CHANGELOG.md]: https://github.com/cerner/terra-core/pull/2749/commits/48cd5b0cb80a9c6069e7621b18e9f1d7fdf68fd0 fixed here

File: packages/terra-status-view/src/terra-dev-site/test/status-view/StatusViewTestCommon.module.scss [hunk 4]
```diff
@@ -6,6 +6,7 @@
 
   .status-view-wrapper-2 {
     border: 1px solid #000;
+    height: 500px;
```
Comment (neilpfeiffer) [inline packages/terra-status-view/src/terra-dev-site/test/status-view/StatusViewTestCommon.module.scss]: What is the reason for adding the fixed height for the test?

**Comment by neilpfeiffer** [comment]
UX +1 

No change to visual layout after the change.
